webapp/src/components/call_widget/component.tsx

```diff
@@ -1208,7 +1220,7 @@ export default class CallWidget extends React.PureComponent<Props, State> {
         const MuteIcon = window.callsClient.isMuted() ? MutedIcon : UnmutedIcon;
         const muteTooltipText = window.callsClient.isMuted() ? 'Click to unmute' : 'Click to mute';
 
-        const hasTeamSidebar = Boolean(document.querySelector('.team-sidebar'));
+        const hasTeamSidebar = Boolean(document.querySelector('.team-sidebar')) || Boolean(this.props.global);
```

Comment:
Is this correct? Has team sidebar if global?  Or, maybe it should be renamed to: `widerWidget` or something, to be clearer?

Reply:
Yes, it's a naming/semantic mismatch. I like your suggestion, thanks!